index.html

```diff
@@ -1,224 +1,331 @@
-            </div>
-            <div class="img-four remaining-img">
-                <h2 class="card-heading-white">Profesional Art Printing Data</h2>
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
-                    industry's standard.
-                </p>
-                <ul class="skills-details">
-                    <li>HTML</li>
-                    <li>Bootstrap</li>
-                    <li>Ruby</li>
-                </ul>
-                <button class="project-details" type="button">See project</button>
-            </div>
-            <div class="img-five remaining-img">
-                <h2 class="card-heading-white">Profesional Art Printing Data</h2>
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
-                    industry's standard.
-                </p>
-                <ul class="skills-details">
-                    <li>HTML</li>
-                    <li>Bootstrap</li>
-                    <li>Ruby</li>
-                </ul>
-                <button class="project-details" type="button">See project</button>
-            </div>
-            <div class="img-six remaining-img">
-                <h2 class="card-heading-white">Profesional Art Printing Data</h2>
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
-                    industry's standard.
-                </p>
-                <ul class="skills-details">
-                    <li>HTML</li>
-                    <li>Bootstrap</li>
-                    <li>Ruby</li>
-                </ul>
-                <button class="project-details" type="button">See project</button>
-            </div>
-            <div class="img-seven remaining-img">
-                <h2 class="card-heading-white">Profesional Art Printing Data</h2>
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
-                    industry's standard.
-                </p>
-                <ul class="skills-details">
-                    <li>HTML</li>
-                    <li>Bootstrap</li>
-                    <li>Ruby</li>
-                </ul>
-                <button class="project-details" type="button">See project</button>
-            </div>
+          </div>
         </div>
+      </div>
+    </section>
+    <section class="card-section">
+      <div id="hover-content" class="cards rel-pos">
+        <div id="img-hover" class="img rel-pos">
+          <button class="project-details btn-pos" type="button">
+            See project
+          </button>
+        </div>
+        <div id="dark-hover" class="img img-one">
+          <h1>Profesional Art Printing Data</h1>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
```

Comment:
- [OPTIONAL] You have done a great job so far, you did great implementing the hover as requested by the reviewer, But here is one important aspect you can still improve on. 😄 👍 👍
- - As explained by the previous reviewer, you need to implement the hover state on each of your project cards. By default in the desktop view, the See Project button is hidden, but when a user hovers over a project, the See project button should appear on the bottom of the project card.
Currently, you only implemented this on the first card, but this should be implemented on all project cards. 👍 Please make sure you fix this.
- - Here is a video from a reviewer that best explains this
VIDEO: 

https://user-images.githubusercontent.com/107632780/229279970-44d034c5-9fbe-4478-8e26-d596f0335c67.mp4

Reply:
Thank you so much, I will fix it!